### PR TITLE
refactor(client): space between passed icons and their sibling

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #48101

I have changed this to Inline because its usage don't revolve around this attribute (the newer section), but the legacy section does use it as spacer. 

Instead of refactoring it, I have added this as a quick hack, because these section may be deleted with the legacy section 

Why I didn't add it in RTL layout PR? because I may be missing a point or a section but so far they are looking the same.

image

### CODE REVIEW

**Comment by Sboonny** [comment]
Here is a similar issue 

image

File: client/src/templates/Introduction/intro.css [hunk 1]
```diff
@@ -426,8 +427,10 @@ button.map-title {
   padding: 25px;
 }
 
+/* this margin is used to center the element. 
+ToDo: find out why, and remove the need for it */
 .block-grid .map-title > svg {
-  margin: 10px;
+  margin-bottom: 4px;
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Introduction/intro.css:433]: Not sure about this. We can do margin: 4px if you wanna save space.

Comment (ahmadabdolsaheb) [inline client/src/templates/Introduction/intro.css:433]:
```suggestion
  margin: 4px;
```

Comment (Sboonny) [reply client/src/templates/Introduction/intro.css:433]: ya this one is weird, I am using it to center the icon with the title in `/learn/{courseSlug}`, the test below happened in `/learn/2022/responsive-web-design/`


with `margin-bottom: 4px`
image


Here how it looks with `margin: 4px`

image

Comment (Sboonny) [reply client/src/templates/Introduction/intro.css:433]: It's a property in the font itself, it has margin-bottom built in it?

disabling font property in this selector fix the weird alignment

```css
button, input, optgroup, select, textarea {
    color: inherit;
    /* font: inherit; */
    margin: 0;
  ```

Comment (ahmadabdolsaheb) [reply client/src/templates/Introduction/intro.css:433]: Sure, we could leave it as you have it.

**Review by ahmaxed — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**